packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/main/index.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Set up self-service registration
+meta:
+  - name: description
+    content: Okta's self-service registration lets you configure a custom app or the Okta homepage for use when users self-register.
+layout: Guides
+---
+
+This article shows you how to set up Okta's self-service registration (SSR) functionality, which lets you configure a custom app or the Okta homepage to allow users to self-register.
+
+---
+
+**Learning outcomes**
+
+* Use policies to govern SSR behavior.
+* Enable and disable SSR recovery options.
+* Configure SSR options in the Okta Sign-in Widget.
+
+**What you need**
+
+* This is an Early Access feature. To enable it, contact [Okta Support](https://support.okta.com/help/s/?_ga=2.17747641.1660906902.1597076228-1076744453.1575496867).
+* [Super admin permissions](https://help.okta.com/okta_help.htm?id=ext_superadmin) are required to enable self-service registration.
+* If you are hosting your own [customized Widget](https://developer.okta.com/code/javascript/okta_sign-in_widget/), version 2.9 or later is required.
+
+**Sample code**
+
+n/a
+
+---
+
+## Overview
+
+A new user's first impression starts with the registration process, making it a critical aspect to get right. Because you're asking a user to give you information for the first time, you need to personalize and streamline the experience.
+
+Okta's self-service registration lets you configure a custom app or the Okta Homepage for use when users self-register. After you configure and enable your self-service registration policy, a sign-up link appears in the [Okta Sign-In Widget](/code/javascript/okta_sign-in_widget/). Users who click this link are directed to a Create Account registration form that is based on your registration policy.
```

Comment:
Create Account registration form > Create Account **_R_**egistration form?
Please double-check the form name. Should be uppercase "R".